<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.2.2 (Python 3.13.3 on win32)" generated="2025-05-14T17:59:01.088409" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="c:\Users\Hanane.BEN-AZIZE\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2025-05-14T17:59:01.155908" elapsed="2.273135"/>
</kw>
<status status="PASS" start="2025-05-14T17:59:01.155238" elapsed="2.274345"/>
</test>
<status status="PASS" start="2025-05-14T17:59:01.092810" elapsed="2.338649"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat name="Robot Interactive Console" id="s1" pass="1" fail="0" skip="0">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2025-05-14T17:59:01.074788" level="WARN">Error in file 'c:\Users\Hanane.BEN-AZIZE\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
